src/tasks/buildTaskProvider.ts

```diff
@@ -126,7 +136,12 @@ export class BuildTaskProvider extends CommandTaskProvider {
 
 			return helper.resolveAppBuildCommandLine(context, task.definition.titaniumBuild);
 		} else if (definition.titaniumBuild.projectType === 'module') {
+
 			definition.titaniumBuild.projectDir = path.join(definition.titaniumBuild.projectDir, definition.titaniumBuild.platform);
+
+			if (!definition.titaniumBuild.target) {
```

Comment:
I think this (and the other prompts for target/device related information) should maybe be gated dependent on the SDK version used (which should be whatever version the SDK PR ships in). Otherwise there's a potential for issues to come in around why this feature isn't working

Reply:
> I think this (and the other prompts for target/device related information) should maybe be gated dependent on the SDK version used (which should be whatever version the SDK PR ships in). Otherwise there's a potential for issues to come in around why this feature isn't working

Ha yes, there are a cupple of otpions : 
 - hide the buttons when SDK is too old
 - add a "Error/Warning message" when the SDK is too old but build the module and stop when application build start 
 - add a "Error/Warning message" and stop immediatly when the SDK is too old before building the module
 - let both build as it but add an INFO message saying that SDK should be Upgrade
 -

Reply:
What about an extension of your first bullet point, and maybe instead fold in https://github.com/tidev/vscode-titanium/pull/998 for when the selected SDK is older? If we land this PR I can rework that one on-top of this to reflect that?

It would still expose a potential issue if someone configures a target via their own tasks.json, but I think the chance of someone doing that and not knowing the restrictions is small

Reply:
yes, sure it is a good idea. Murphy will always be around ;)

Reply:
Hi @ewanharris, so what do we do now with this PR ?

(I have noticed something annoying with Android emulator : a command windows is opened (and not automaticaly closed), this is not in anyway blocking or anything else but I dont like it and it make user experience a little worse, we may adress this little issue too in this PR)

Sorry I was away of open source project for a while, I just get back on track

Reply:
Hey @DzzD, I'll need to look back over this. I see I left some comments initially but I can either merge and push commits for those or just push directly to your branch before merge to get them resolved.